requirements/main.in

```diff
@@ -55,6 +55,7 @@ readme-renderer[md]>=36.0
 requests
 requests-aws4auth
 redis>=2.8.0,<5.0.0
+rich
```

Comment:
I _think_ we could replace most, if not all, of the `rich` usage with `click` builtins that we already use. See:

https://click.palletsprojects.com/en/8.1.x/utils/#printing-to-stdout
https://click.palletsprojects.com/en/8.1.x/api/#click.echo

https://click.palletsprojects.com/en/8.1.x/utils/#ansi-colors
https://click.palletsprojects.com/en/8.1.x/utils/#intelligent-file-opening

Reply:
[b075088](https://github.com/pypi/warehouse/pull/13615/commits/b0750885fde3dc9edadf2c826830c572e81c6e27)